src/web/jdv/server/moos_messages.py

```diff
@@ -70,11 +24,14 @@ def get_moos_messages(log_filenames, t_start, t_end) -> str:
     csv_writer = csv.DictWriter(output_string_io, fieldnames=fieldnames)
     csv_writer.writeheader()
 
+    t_start = t_start or 0
```

Comment:
You make a good point, but the only difference is if the caller wanted to pass None (no start date) for t_start, and also pass something in for t_end.  I suppose they could pass in 0 for t_start instead of None, but I felt the "None" makes it more explicit that they don't want any start time, just an end time.  I may be overthinking this, which is typical!

Reply:
I didn't consider that. Yeah it gets tricky because there is sort of an implicit conversion from a start date to an index. I would just go for what you think is most clear and include a note about that in the doc string